src/store/modules/bot-commands/utils/extractCommandsFromMessages.ts

```diff
@@ -0,0 +1,25 @@
+import { BotCommand } from '@/store/modules/bot-commands/types'
+import { NormalizedChatMessageTransaction } from '@/lib/chat/helpers'
+
+/**
+ * The function retrieves all bot commands from the message array
+ * @param {string} recipientId  Message recipient address
+ * @param {NormalizedChatMessageTransaction[]} messages Array of normalized messages
+ * @return {BotCommand[]} Bot commands array
```

Comment:
```suggestion
 * @param recipientId - Message recipient address
 * @param messages - Array of normalized messages
 * @return Bot commands array
```

You can remove the types from docblock. They are already described by TS.
https://tsdoc.org/